### PR TITLE
Docs deploying

### PR DESCRIPTION
**Pull request by jonnilundy**
Beginning work on Deploying section.

- Updated How It Works page
- Updated tutorials links to add categories
- Added Deploying page with Instant Deployment, Continuous Deployment, and links to tutorials

Questions:
- Should we create a new doc category for deploying?
- How should we document deploying more advanced projects (ie. ones with multiple services)

### CODE REVIEW

**Comment by jonnilundy** [comment]
#157

File: ui/src/pages/docs/intro/deploying.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+## Create project
+
+Now that you have our local files ready, it's time to go to the [WeDeploy Dashboard](http://dashboard.wedeploy.com) and create a new project (and if you haven't already, a WeDeploy account).
+
+To do this, open the [Dashboard](http://dashboard.wedeploy.com) and click "New Project" in the top right corner. Then give you project a name and click "Create Project".
+
+![create new project](/images/docs/intro/deploying--create-project.png)
+
+Great! Now you're project is ready for you to deploy your local files to it.
```
Comment (NolanChan) [inline ui/src/pages/docs/intro/deploying.md]: you're => your

Comment (jonnilundy) [inline ui/src/pages/docs/intro/deploying.md]: Good catch! 👍

File: ui/src/pages/docs/intro/deploying.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+1. Open your terminal to the project you prepared above and commit any changes you made. `git add . && git commit -m "prepare project for WeDeploy"`
+2. Push changes to your GitHub repo. `git push origin master`
+
+Perfect, now your GitHub repo is ready to be deployed. Next, you must **connect that repo to your WeDeploy project**.
+
+1. On the [Dashboard](http://dashboard.wedeploy.com), go to your new project and click on "Deployment"
+2. Choose the GitHub method
+3. Login to your GitHub Account and then select the repo and branch that you just pushed to
+4. Click "Connect Repository" and then go to "Overview" on top left of your screen. You will see the deployment automatically start.
```
Comment (NolanChan) [inline ui/src/pages/docs/intro/deploying.md]: sounds a bit smoother to say 'then go to "Overview" on _the_ top left of your screen...'

File: ui/src/pages/docs/intro/deploying.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Now the moment you've been waiting for. With your terminal open to the project you want to deploy, just run `we deploy` and it's live! Follow the prompts in your terminal in order to open your new project url in any browser.
+
+</article>
+
+<article id="2">
+
+## Continuous Deployment
+
+Continuous Deployment is powered by our handy [GitHub](htps://github.com) integration that triggers a deploy on your project everytime you push changes to the designated repo branch. It is super easy to setup and greatly speeds up your development flow.
```
Comment (NolanChan) [inline ui/src/pages/docs/intro/deploying.md]: everytime => every time

File: ui/src/pages/docs/intro/deploying.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+##### 1. Setup project
+
+To Start, create a new folder wherever you want to your project to live. Inside that folder, you must create a new file and name it `container.json`. This is the primary config file that you will use in all your WeDeploy projects to designate what service you want to deploy (and some other [fun configurations](/docs/intro/configuration-files.html)).
```
Comment (NolanChan) [inline ui/src/pages/docs/intro/deploying.md]: To Start, create a new folder wherever you want to your project to live. => To start, create a new folder wherever you want your project to live.

File: ui/src/pages/docs/intro/deploying.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+```
+{
+	"id": "ui",
+	"type": "wedeploy/hosting:latest"
+}
+```
+
+##### 2. Add files and push
+
+So you're empty project is ready to deploy, but we want to make it more interesting by **adding our own static files** for WeDeploy to serve.
```
Comment (NolanChan) [inline ui/src/pages/docs/intro/deploying.md]: you're => your

**Pull request closed without merging**